modules/heat_conduction/src/meshgenerators/PatchSidesetGenerator.C

```diff
@@ -399,6 +400,8 @@ PatchSidesetGenerator::boundaryElementHelper(MeshBase & mesh, libMesh::ElemType
       return mesh.add_elem(new libMesh::Quad8);
     case 7:
       return mesh.add_elem(new libMesh::Quad9);
+    case 33:
```

Comment:
Do we need the switch statement here?  `return mesh.add_elem(libMesh::Elem::build(type));` should cover it, no?  We could test for `Elem::dim() < 3` if we want to keep the protection that `mooseError` gives.

Reply:
changed in https://github.com/idaholab/moose/pull/25742